api-doc.md

```diff
@@ -417,3 +417,71 @@
     - Headers : none
     - Message : Acceptence mail has been sent successfully
 ```
+
+### 9. Get Career Page LifeatCanopas Section Images
+
+```
+- Method : GET
+- Endpoint : /api/lifeimages
+- Description : API for get images of lifeat canopas section of career page
+- Request:
+    - Headers : none
+    - Body : none
+
+- Response :
+
+    - If API is not found or image data not found then,
+    - Status Code: 404 not found
+    - Data: no response data
+
+    - If any server error is there then,
+    - Status Code: 500 Internal server error
+    - Data: no response data
+
+    - If request will success ,
+    - Status Code: 200 Ok
+    - Headers : none
+    - Data :
+        [
+            {
+                 "id": 1,
+                 "image_urls": "https://canopas-website.s3.ap-south-1.amazonaws.com/lifeCanopas/1-400.webp,https://canopas-website.s3.ap-south-1.amazonaws.com/lifeCanopas/1-800.webp,https://canopas-website.s3.ap-south-1.amazonaws.com/lifeCanopas/1-1600.webp",
+                 "index": 0
+            },
+        ]
+
+```
+
+### 10. Get Career Page Perks and Benefit Section Images
+
+```
+- Method : GET
+- Endpoint : /api/perksimages
+- Description : API for get images of perks and benefit section of career page
+- Request:
+    - Headers : none
+    - Body : none
+
+- Response :
+
+    - If API is not found or image data not found then,
+    - Status Code: 404 not found
+    - Data: no response data
+
+    - If any server error is there then,
+    - Status Code: 500 Internal server error
+    - Data: no response data
+
+    - If request will success ,
+    - Status Code: 200 Ok
+    - Headers : none
+    - Data :
+        [
+            {
+                 "id": 1,
+                 "image_urls": "https://canopas-website.s3.ap-south-1.amazonaws.com/perksBenefits/1-400.webp,https://canopas-website.s3.ap-south-1.amazonaws.com//perksBenefits/1-800.webp",
```

Comment:
why we are returning images as a single string?

Reply:
because it is suggested by sumita

Reply:
yaa I see that you are storing images in a single field but then you need to separate out it on the front end right?
so why you are not separating out here and sending an array of images?

Reply:
but i am separating out in js where i am getting this images

Reply:
ok that's fine but backend is more faster than frontend and also it can be reusable. so i just suggest this way.